list.json

```diff
@@ -9,5 +9,16 @@
       "min": 1,
       "max": 1
     }
+  },
+  {
+    "name": "philips-hue-adapter",
+    "display_name": "Philips Hue",
+    "description": "Philips Hue color light support",
+    "version": "0.3.0",
+    "url": "https://github.com/mozilla-iot/philips-hue-adapter/releases/download/0.3.0/philips-hue-adapter-0.3.0.tgz",
+    "api": {
+      "min": 1,
+      "max": 1
+    }
```

Comment:
When we merge these, it would probably be good to keep the list in alphabetical order by name.